src/evaluators/response/PHAL_ResponseSquaredL2DifferenceSide_Def.hpp

```diff
@@ -70,6 +100,9 @@ ResponseSquaredL2DifferenceSideBase(Teuchos::ParameterList& p, const Teuchos::RC
 
   this->setName("Response Squared L2 Error Side" + PHX::print<EvalT>());
 
+  if(plist->isParameter("Response Depends On Solution Column") && plist->get<bool>("Response Depends On Solution Column"))
```

Comment:
Here too: plist is not const, so you can use the get method providing a default, and make this line shorter.

Reply:
same as above